changelog.html

```diff
@@ -55,7 +55,9 @@
 <!-- Record your changes in the trunk here. -->
 <div id="trunk" style="display:none"><!--=TRUNK-BEGIN=-->
 <ul class=image>
-  <li class=>
```

Comment:
It's better if you leave this part to the person merging the PR. It's too easy for changelog changes in the PR to prevent automatic merges.